docs/static/settings/configuration-management-settings.asciidoc

```diff
@@ -78,3 +78,19 @@ the client’s certificate.
 `xpack.management.elasticsearch.ssl.keystore.password`::
 
 Optional setting that provides the password to the keystore.
+
+`xpack.management.elasticsearch.cloud_id`::
+
+If you're using Elasticsearch cloud, you should specify the identifier here.
+This is an alternative to specifying `xpack.management.elasticsearch.hosts`,
+if `cloud_id` is set having also a `hosts` configuration makes no sense.
```

Comment:
I might consider rewording this to something like "If `cloud_id` is set, `hosts` must not be configured. @karenzone - thoughts?